PalasoUIWindowsForms/ImageToolbox/PalasoImage.cs

```diff
@@ -7,15 +7,15 @@
 using Palaso.Code;
 using Palaso.IO;
 using Palaso.UI.WindowsForms.ClearShare;
-using System.Linq;
 
 namespace Palaso.UI.WindowsForms.ImageToolbox
 {
 	public class PalasoImage : IDisposable
-
 	{
 		private Metadata _metadata;
 
+		private string LeakedFilePath { get; set; }
```

Comment:
I'm not a fan of the variable name. It does follow from the local variable that you are exposing through the out parameter, but could you name it TemporaryFilePath or something else?